src/models/Rcs.ts

```diff
@@ -56,6 +56,11 @@ export class AMTConfiguration {
   wifiConfigs?: ProfileWifiConfigs[]
 }
 
+export class ProfilesWithCnt {
+  profiles: any
```

Comment:
we can type this right? at the very least we know its an array

Reply:
done